### PR TITLE
Update to model 0.0.48

### PR DESCRIPTION
**Pull request by igoihman**
The more relevant changes in the new version of the model are the
following:

- Fix `OpenID` attributes.
- Add Cluster API listening method.

### CODE REVIEW

**Comment by igoihman** [comment]
[test]

**Comment by igoihman** [comment]
@tzvatot @zgalor @vkareh PTAL

File: clustersmgmt/v1/cluster_api_type.go [hunk 1]
```diff
@@ -55,6 +57,29 @@ func (o *ClusterAPI) GetURL() (value string, ok bool) {
...
 
+// Listening returns the value of the 'listening' attribute, or
+// the zero value of the type if the attribute doesn't have a value.
+//
+// The listening method of the API server.
+func (o *ClusterAPI) Listening() ListeningMethod {
+	if o != nil && o.listening != nil {
+		return *o.listening
+	}
+	return ListeningMethod("")
```
Comment (zgalor) [inline clustersmgmt/v1/cluster_api_type.go]: I am not familiar with the ocm-sdk logic, but is that what we usually do? No default to one of the methods?
If it were a bool, we would probably default to false.

File: clustersmgmt/v1/listening_method_list_type_json.go [hunk 2]
```diff
@@ -26,46 +26,47 @@ import (
 	"github.com/openshift-online/ocm-sdk-go/helpers"
 )
 
-// MarshalOpenIDURLsList writes a list of values of the 'open_IDURLs' type to
+// MarshalListeningMethodList writes a list of values of the 'listening_method' type to
```
Comment (zgalor) [inline clustersmgmt/v1/listening_method_list_type_json.go]: why is that a list?

File: clustersmgmt/v1/listening_method_type.go [hunk 3]
```diff
@@ -0,0 +1,30 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
```
Comment (zgalor) [inline clustersmgmt/v1/listening_method_type.go]: 2020 :)

File: clustersmgmt/v1/cluster_api_type.go (hunk 1 above)
Comment (igoihman) [reply clustersmgmt/v1/cluster_api_type.go]: it's a generated code by `make generate`. The defaults will be handled in clusters-service code

File: clustersmgmt/v1/listening_method_type.go (hunk 3 above)
Comment (igoihman) [reply clustersmgmt/v1/listening_method_type.go]: again, it's generated, the scripts need an update :)

**Pull request merged**